### DISCUSSION TITLE
feature: Variable acting on Account level

### DISCUSSION THREAD

**Discussion by worldofjimmy** [AscensionGameDev/Intersect-Engine#1543] [Ideas]
**Is your feature request related to a problem? Please describe.**
I have made a tutorial in my game for newly created players. But the tutorial would be skipable if one player on the account already completed it. I have tried this with player and global variable but I can't find solution made via them.

I think that if this would be doable if there was variable acting on account level. Otherwise new players might skip the tutorial by mistake. Account variable would also open up many features for e.g. player housing that all players on the same account has access to enter the house/area. Also another purpose for variable on account level would be another dimension of quest factions/achievements.

**Describe the solution you'd like**
An addition to the existing player and global variables - probably named account variables.

**Additional context**
![image](https://user-images.githubusercontent.com/54074317/161392027-08187371-5346-4beb-85dd-cb6875204c83.png)

**Comment by dudevelopement** [comment]
That would be good, because with that we can make a VIP system that counts the days with the entire player account and not just by character.

**Comment by Uyarrr** [comment]
Really good idea.
